packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -206,7 +206,7 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       <FocusZone direction={FocusZoneDirection.bidirectional} {...focusZoneProps}>
         <MarqueeSelection selection={selection} isEnabled={true}>
           <SelectionZone selection={selection} selectionMode={SelectionMode.multiple}>
-            <div className={css('ms-BasePicker-text', classNames.pickerText)} role={'list'}>
+            <div className={css('ms-BasePicker-text', classNames.pickerText)} role="listitem">
```

Comment:
Semantically `list` seems correct here since that's what this is (it's not an individual item). There's probably some other better way to address the flagged issue.

Reply:
will revert this and see if there is a better way of addressing the accessibility issues which arise due to this.